examples/print_styles.cu

```diff
@@ -83,6 +99,32 @@ int main([[maybe_unused]] int argc, [[maybe_unused]] char **argv)
   print(A4);
   // example-end print-example-1
 
+  // create stream
```

Comment:
I would leave out all this code since it's not really relevant to the fix, and we don't want to encourage people to use float3 types in tensors.

Reply:
Done.